numpy/core/src/multiarray/einsum.c.src

```diff
@@ -341,6 +369,25 @@ finish_after_unrolled_loop:
         /* Finish off the loop */
         goto finish_after_unrolled_loop;
     }
+#elif EINSUM_USE_NEON_64 && @float64@
+        /* Unroll the loop by 8 */
+        while (count >= 8) {
+            count -= 8;
+
+/**begin repeat2
+ * #i = 0, 2, 4, 6#
+ */
+            a = vmulq_f64(vld1q_f64(data0+@i@), vld1q_f64(data1+@i@));
```

Comment:
replace these intrinsics with:
`vld1q_f64` -> `npyv_load_f64`
`vmulq_f64` -> `npyv_mul_f64`
`vaddq_f64` -> `npyv_add_f64`
`vst1q_f64` -> `npyv_store_f64`

for more intrinsics, you have to search or dive in the [NPYV](https://github.com/numpy/numpy/tree/master/numpy/core/src/common/simd)

Reply:
BTW why you didn't use FMA3 here, well currently there's no mul-add intrinsic in NPYV but I guess It will not be that hard to impmlent it.

Reply:
I will make another PR for mul-add intrinsic in NPYV.